management-account/terraform/sso-scim.tf

```diff
@@ -8,3 +8,11 @@ module "scim" {
   sso_identity_store_id = local.sso_admin_identity_store_id
   not_dry_run           = true
 }
+
+module "entraid_scim" {
+  # tflint-ignore: terraform_module_pinned_source
+  source              = "github.com/ministryofjustice/moj-terraform-scim-entra-id"
```

Comment:
I was planning on doing some work to wrangle all the modules the root account is dependent on 🌴 And I was thinking that the working group (and eventually Modernisation Platform) will need to own and administrate all of these repositories 🔑 

So pre-emptively, could you add the aws-root-account-admin-team as admins to this repo as well please 🙏

Reply:
`aws-sso-admin` are already admins in this repo. Is that not sufficient?

Reply:
Regardless: once this PR is approved and merged, the team will be added: https://github.com/ministryofjustice/data-platform-github-access/pull/185

Reply:
We currently have `aws-root-account-admin-team` that links the working group to its relevant access and responsibilities 🔐

So it should be that team, as when we start onboarding MP Engineers into Root Account responsibilities, it'd be preferable to just have the one team to worry about 🙈